scripts/src/public/services/migration.ts

```diff
@@ -27,18 +29,12 @@ export const accountStatus = async function(req: express.Request, res: express.R
 
 	const app_blockchain_version = await getAppBlockchainVersion(appId);
 
-	const isBurnedRes = await client.get(`${ getConfig().migration_service }/accounts/${ publicAddress }/status`, {
```

Comment:
Remove the creation of `client`, axios, axios-retry and everything else that becomes irrelevant or unused after this edit.